tests/integration/view/one_to_many/simple_test.go

```diff
@@ -402,25 +402,26 @@ func TestView_OneToManyWithDoubleSidedRelation_Errors(t *testing.T) {
 					}
 				`,
 			},
-			testUtils.CreateView{
-				Query: `
-					AuthorView {
-						name
-						books {
+			/*
```

Comment:
question: did you forget to uncomment or remove this?

Reply:
I did! Thanks Keenan

- [ ]  Uncomment test, and revert the second half of the test